esmvaltool/cmorizers/data/formatters/datasets/ndp.py

```diff
@@ -35,7 +35,7 @@ def _extract_variable(cmor_info, attrs, var_file, out_dir, cfg):
     array = grid_file.ReadAsArray()
     for missing_value in cfg['missing_values']:
         array = np.ma.masked_equal(array, missing_value)
-    array = array.astype(np.float)
+    array = array.astype(np.float64)
```

Comment:
```suggestion
    array = array.astype(np.float32)
```
is most commonly used in CMIP data